### PR TITLE
Changes for issue #441.

### PR DESCRIPTION
**Pull request by manish-in-java**
Added code to allow displaying a different label for file selection controls that support multiple file selection.

### CODE REVIEW

**Comment by shehi** [comment]
Please also update demos to reflect your changes, adding samples with multiple file selection. Once confirmed, I will merge. Do so for all the themes provided in demos. Thanks.

File: src/js/jquery.uniform.js [hunk 1]
```diff
@@ -610,11 +610,15 @@
                 apply: function ($el, options) {
                     var ds, $div, $filename, $button;
 
+                    // Issue# 441: Check if the control supports multiple selection.
```
Comment (shehi) [inline src/js/jquery.uniform.js]: Fix as "#441" please. If you can, squash your commits into a single commit and update your PR.

**Comment by manish-in-java** [comment]
Demos updated. Contributor license agreement signed.

**Comment by shehi** [comment]
When you are done, please don't forget to sign CLA.

**Comment by shehi** [comment]
Ok, so generally, this PR only makes it possible to have a label for multiple uploads field with word in plural case. Merging... I will update dist files later on and make a release.

**Pull request merged**